### DISCUSSION TITLE
졸업 후에는 어떻게 되나요?

### DISCUSSION THREAD

**Discussion by byulla** [2022-IT-SHOW-Shade/android#56] [Q&A] [answered]
졸업 후에는 학교 변경(ex. 중학교 -> 고등학교)을 어떻게 해야 하는 지 알 수 있을까요?
어떤 기능을 사용하는지 알고 싶습니다! 😊

**Comment by 0pyaq0** [comment, marked as answer]
마이페이지에 들어가셔서 [학교 변경] 메뉴를 클릭하신 후 원하시는 학교를 검색하여 선택하시면 자동으로 변경됩니다!
감사합니다! 😊
